### PR TITLE
Files to manage the content of the CSV metadata header

### PR DESCRIPTION
**Pull request by xhoenner**
Those files will be updated periodically. Related to https://github.com/aodn/harvesters/pull/585

### CODE REVIEW

File: PARAMETERS_MAPPING/unit_view.csv [hunk 1]
```diff
@@ -0,0 +1,61 @@
+id;name;short_name;definition;date_added
+451;Bytes;Bytes                                                                                                         ;;2013-07-18
+769;Cells per millilitre;Cells ml-1                                                                                                    ;;2014-07-10
+445;Decibars;dBar                                                                                                          ;;2013-07-18
+439;Decibels;dbel                                                                                                          ;;2013-07-18
+440;Degrees;deg                                                                                                           ;;2013-07-18
+429;Degrees Celsius;degC                                                                                                          ;;2013-07-18
+430;Degrees Kelvin;degK                                                                                                          ;;2013-07-18
```
Comment (mhidas) [inline PARAMETERS_MAPPING/unit_view.csv]: Kelvin are not degrees. Just Kelvin.

Comment (xhoenner) [reply PARAMETERS_MAPPING/unit_view.csv]: Changed corresponding file accordingly, thanks @mhidas

File: PARAMETERS_MAPPING/unit_view.csv [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+512;Microatmospheres;uatm                                                                                                          ;;2013-10-08
+588;Micrograms per litre;ug l-1                                                                                                        ;;2014-01-15
+487;Micromoles per kilogram;umol kg-1                                                                                                     ;;2013-09-09
+447;Micromoles per litre;umol l-1                                                                                                      ;;2013-07-18
+585;Micromoles per second per square metre;umol s-1 m-2                                                                                                  ;;2014-01-08
+486;Microwatts per square centimetre per nanometre;uW cm-2 nm-1                                                                                                  ;;2013-09-09
+511;Millibars;mBar                                                                                                          ;;2013-10-01
```
Comment (bpasquer) [inline PARAMETERS_MAPPING/unit_view.csv]: mBar -> mbar. see above

File: PARAMETERS_MAPPING/unit_view.csv [hunk 3]
```diff
@@ -0,0 +1,61 @@
+id;name;short_name;definition;date_added
+451;Bytes;Bytes                                                                                                         ;;2013-07-18
+769;Cells per millilitre;Cells ml-1                                                                                                    ;;2014-07-10
+445;Decibars;dBar                                                                                                          ;;2013-07-18
+439;Decibels;dbel                                                                                                          ;;2013-07-18
```
Comment (bpasquer) [inline PARAMETERS_MAPPING/unit_view.csv]: Decibars should be abbreviated "dbar"
and the abbreviation for decibel is "dB"

unit symbol are lower case unless the unit is derived from a proper name

Comment (xhoenner) [reply PARAMETERS_MAPPING/unit_view.csv]: Thanks @bpasquer, those have been fixed in https://github.com/aodn/data-services/pull/793/commits/1f1d46e36ffe556b6d3c21f8f4af576acfa046fc

**Comment by bpasquer** [comment]
I am absolutely delighted

**Pull request merged**